src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -69,7 +70,7 @@ public function getItemFromIri(string $iri, array $context = [])
             return $item;
         }
 
-        throw new InvalidArgumentException(sprintf('Item not found for "%s".', $iri));
+        throw new NotFoundHttpException(sprintf('Item not found for "%s".', $iri));
```

Comment:
A `NotFoundHttpException` should not be throw here (we can use this class in a non HTTP context e.g. from the command line). You should probably introduce a new exception extending `InvalidArgumentException` for BC.